### ISSUE TITLE
People picker selects first item when keyboard user tabbing through

### ISSUE THREAD

**Issue by BrendanMcK** [OfficeDev/office-ui-fabric-react#5915]
### Bug Report 
 
- __Package version(s)__: current as of 8/12/2018
- __Browser and OS versions__: any
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) No
 
Requested priority: (Blocking, High, Normal, Low)  Normal, but potential a11y impact
 
Products/sites affected: (if applicable) Unknown
 
#### Describe the issue:

Go to people picker page here: https://developer.microsoft.com/en-us/fabric#/components/peoplepicker

Tab to the people picker. 
 
#### Actual behavior:

When the people picker receives focus, a popup appears with the initial item selected.
Hitting tab will end up selecting this item, and dismissing the popup
It is only when tab is hit again that focus moves to the next control.

 
#### Expected behavior:
 
A keyboard user should be able to tab through and shift-tab backwards though any control without having unexpected side-effects. If a keyboard user is tabbing from some control before the people picker to some control after, in tabbing through, they can end up inadvertently selecting an additional item.

To avoid this, the user may have to reverse tab and delete the extra item; or when tabbing forwards, use ESC to dismiss the popup before moving onwards. This is not always easy or obvious when rapidly tabbing around a page. 

Perhaps the ideal situation here is that the popup should initially appear with no item selected; and in this no-selection state, tab should dismiss it and move focus to the next control. Tab then only selects the current item if the user has use the arrow keys or typed in text to trigger an intentional selection.

### If applicable, please provide a codepen repro:
 
See repro above on Fabric components page.

**Comment by JasonGore** [comment]
I spoke with Jon and he's working on a fix.
